soh/src/overlays/actors/ovl_En_Ishi/z_en_ishi.c

```diff
@@ -330,6 +330,10 @@ void EnIshi_Init(Actor* thisx, GlobalContext* globalCtx) {
         Actor_Kill(&this->actor);
         return;
     }
+    // remove fire temple blocking rocks for child
+    if (gSaveContext.n64ddFlag && globalCtx->sceneNum == 0x061 && this->actor.params == (-255)) {
```

Comment:
do we want to do this always in rando? or just when doing entrance rando?

Reply:
This, deku tree, and gtg gate are all checking for dungeon entrance setting on 3ds rando. These are hold overs from the original entrance work, I can add those checks for them here.